op-challenger/fault/responder.go

```diff
@@ -105,7 +105,6 @@ func (r *faultResponder) buildStepTxData(stepData StepCallData) ([]byte, error)
 	return r.fdgAbi.Pack(
 		"step",
 		big.NewInt(int64(stepData.StateIndex)),
-		big.NewInt(int64(stepData.ClaimIndex)),
```

Comment:
I believe this is incorrect - the first argument into the step function is the claim index, not the state index.